src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -2212,3 +2212,9 @@ services:
     autowire: true
     autoconfigure: true
     public: false
+
+  PrestaShopBundle\Form\Admin\Type\ReorderPositionsButtonType:
+    parent: 'form.type.translatable.aware'
+    public: true
+    tags:
+      - { name: form.type }
```

Comment:
I think this is not useful as 
- parent is no more needed with autowrigin
- this type is covered by service discovery
- tag will come from autoconfigure

Reply:
In theory I agree with your but sadly, and after some tests, this conf is the only way to make this Type work.
Do you have some hints to share? :D

Reply:
It's on 8.1 so you may need to add autowire and autoconfigure attributes.

You also may need what I did on this file on the develop branche, check the binding section the $locales variable.